start_swarm_node.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+if [[ $# -ne 1 ]]; then
+  echo "Usage: $0 <data-directory>"
+  exit 1
+fi
+
+DATADIR="$1"
+
+if [[ ! -e $DATADIR/keystore ]]; then
+    mkdir -p $DATADIR
+    echo 'secret' > $DATADIR/password
+    /app/bin/geth --datadir $DATADIR account new --password $DATADIR/password
+fi
+
+which jq
+if [ $? -eq 0 ]
+then
+    KEY=$(jq --raw-output '.address' $DATADIR/keystore/*)
+else
+    printf "\n\nERROR: jq is required to run the startup script\n\n"
+    exit 1
+fi
+
+/app/bin/swarm \
+    --datadir $DATADIR \
+    --password $DATADIR/password \
+    --verbosity 4 \
+    --bzzaccount $KEY \
+    --ens-api '' \
+    --bzznetworkid 922 \
```

Comment:
Is this a specific network? I think we should make sure our node doesn't connect to any public one.

Reply:
Yeah, I removed that now, good catch.